src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -2,39 +2,33 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Diagnostics;
 using System.IO;
 using System.Reflection;
 using System.Resources;
 using System.Runtime.ExceptionServices;
 using Xunit;
-using Xunit.NetCore.Extensions;
 
 namespace System.Tests
 {
     // No appdomain in UWP or CoreRT
     [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "dotnet/corefx #18718")]
     public class AppDomainTests : RemoteExecutorTestBase
     {
+        private string DestTestAssemblyResolvePath { get; }
+        private string DestTestAssemblyTPAPath { get; }
+
         public AppDomainTests()
         {
-            string sourceTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "AssemblyResolveTests.dll");
-            string destTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "AssemblyResolveTests", "AssemblyResolveTests.dll");
-            if (File.Exists(sourceTestAssemblyPath))
-            {
-                Directory.CreateDirectory(Path.GetDirectoryName(destTestAssemblyPath));
-                File.Copy(sourceTestAssemblyPath, destTestAssemblyPath, true);
-                File.Delete(sourceTestAssemblyPath);
-            }
+            DestTestAssemblyResolvePath = Path.Combine(base.TestDirectory, "AssemblyResolveTests.dll");
+            DestTestAssemblyTPAPath = Path.Combine(base.TestDirectory, "TestAppOutsideOfTPA.exe");
 
-            sourceTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "TestAppOutsideOfTPA.exe");
-            destTestAssemblyPath = Path.Combine(Environment.CurrentDirectory, "TestAppOutsideOfTPA", "TestAppOutsideOfTPA.exe");
-            if (File.Exists(sourceTestAssemblyPath))
+            if (!PlatformDetection.IsNetNative)
```

Comment:
whenever you use this, could you add a comment explaining why it has to be special cased? What doesn't uap/uapaot support here and why isn't that an issue? they are supposed to work the same in general of course.